.github/workflows/lint-docs.yaml

```diff
@@ -8,14 +8,12 @@ on:
   pull_request:
     paths: "docs/**"
```

Comment:
Can you please update the paths here and on line 5 to `'**.md'` instead of `docs/**` ? This will ensure we check all markdown documents, not just the ones in `docs/`. Right now your change above in .github/pull_request_template.md isn't triggering the action because it's not part of docs/.